<launch>

    <include file="$(find crues_control)/robot.launch" />

    <node name="controller" pkg="crues_control" type="path_control.py" required="true">
        <param name="turn_vel" type="double" value="1" />
        <param name="fwd_vel" type="double" value="0.15" />
        <remap from="~rate" to="enc_rate" />
        <param name="path_file" value="$(find crues_control)/paths/map_scan.txt" />
    </node>

</launch>